redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/DatabaseServerV4Endpoint.java

```diff
@@ -0,0 +1,105 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.databaseserver;
+
+import javax.validation.Valid;
+//import javax.validation.constraints.NotNull;
```

Comment:
Remove commented out imports.

Reply:
Disregard, I realize why these are here now.

Reply:
Yeah, there's a lot of stuff that I only commented out, like validation, because maybe we'll need them back soon. There will need to be a cleanup pass later on for anything that stays removed.